src/Character/Controller/BaseController.lua

```diff
@@ -4,20 +4,24 @@ TheNexusAvenger
 Base class for controlling the local character.
 --]]
 
+local THUMBSTICK_INPUT_START_RADIUS = 0.6
+local THUMBSTICK_INPUT_RELEASE_RADIUS = 0.4
 local THUMBSTICK_DEADZONE_RADIUS = 0.2
 
-
+local BLUR_TWEEN_INFO = TweenInfo.new(.25, Enum.EasingStyle.Quad)
 
```

Comment:
Minor styling complaint: `.25` instead of `0.25` being used. This isn't done directly above.